tests/data/pattern_matching_style.py

```diff
@@ -10,6 +10,60 @@
     ): print(2)
     case a: pass
 
+match(
+    arg # comment
+)
+
+match(
+)
+
+match(
+
+
+)
+
+case(
+    arg # comment
+)
+
+case(
+)
+
+case(
+
+
+)
+
+
+match something:
+    case b(): print(1+1)
+    case c(
+        very_complex=True,
+        perhaps_even_loooooooooooooooooooooooooooooooooooooong=-   1
+    ): print(1)
+    case c(
+        very_complex=True,
+        perhaps_even_loooooooooooooooooooooooooooooooooooooong=-1
```

Comment:
Maybe add a trailing comma here just to see that it interacts correctly with the magic trailing comma
```suggestion
        perhaps_even_loooooooooooooooooooooooooooooooooooooong=-1,
```

Reply:
That'll probably need a change in the output as well, will commit separately. Thanks!